node/cn/backend.go

```diff
@@ -528,12 +535,18 @@ func (s *CN) Start(srvr p2p.Server) error {
 	if s.lesServer != nil {
 		s.lesServer.Start(srvr)
 	}
+	if s.governance.ProposerPolicy() == uint64(istanbul.WeightedRandom) {
```

Comment:
Is there any need for a nil check of the `s.governance`?

Reply:
No, there is no need. CN is created with governance